[BZ1836311]: Add a note (#41965)
openshift/openshift-docs: admin_guide/manage_nodes.adoc

----
$ oc get nodes
----

[NOTE]
====

> `$ oc delete node <node>` command does not clean the filesystem of a node. A node can register itself if you do not perform the xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstallation of a node].

Reviewer: * s/`$ oc delete node <node>` command/The `$ oc delete node <node>` command
* As a naive reader, it's not clear what it means for a node to "register itself". Do you mean a node that you have performed `$ oc delete node <node>` on? Register itself with OpenShift?
* Consider "if you do not xref ...blabla.adoc[uninstall the node].

Author: Addressed

> The `$ oc delete node <node>` command does not clean the filesystem of a node. A node which has been deleted, can register itself with OpenShift if you do not perform the xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstall the node].

Reviewer: ```suggestion
The `$ oc delete node <node>` command does not clean the filesystem of a node. A node that has been deleted can register itself again with {product-title} if you do not also xref:../install/uninstalling_cluster.adoc#uninstalling-nodes[uninstall the node].
```

Author: Addressed
